CIP-XXXX/README.md

```diff
@@ -0,0 +1,179 @@
+
+#### 500 Royalty Token Datum
+
+This specification is largely based on [the royalty specification in Nebula](https://github.com/spacebudz/nebula/tree/main#royalty-info-specification), with a couple key departures:
+
+- The royalty token is recommended to be locked at a script address, rather than stored in the user's wallet. This encourages projects to guarantee royalties won't change by sending their royalties to an always-fails (or similar) script address, but still allows for creative royalty schemes and minimizes disruption to existing projects.
+
+- The policyId of the royalty NFT must match that of the reference NFT. This enables lookups based on the user token in the same way as is done for the tokens specified in the original CIP 68 standard.
+
+#### Reference Datum Flag
+
+In addition to providing a way to create guaranteed royalties, this has several advantages:
+
+- Backwards Compatibility - Existing royalty implementations will still work, just not have the same assurances.
+- Minimal Storage Requirement - An optional boolean has about the smallest memory impact possible. This is especially important because it's attached to the - Reference NFT and will be set for each individual NFT.
+- Intra-Collection Utility - This already allows for minting a collection with some NFTs with royalties and some without. A future version of this standard will likely make use of this field to allow for multiple versions of royalties for even more granular control.
+
+
+## Backward Compatibility
+To keep metadata compatibility with changes coming in the future, we introduce a `version` field in the datum.
+
+
+## Path to Active
+
+### Acceptance Criteria
+
+- [ ] This CIP should receive feedback, criticism, and refinement from: CIP Editors and the community of people involved with NFT projects to review any weaknesses or areas of improvement.
+- [ ] Guidelines and examples of publication of data as well as discovery and validation should be included as part of of criteria for acceptance.
+- [ ] Minimal reference implementation making use of [Lucid](https://github.com/spacebudz/lucid) (off-chain), [PlutusTx](https://github.com/input-output-hk/plutus) (on-chain): [Reference Implementation](./ref_impl/)
+- [ ] Implementation and use demonstrated by the community: NFT Projects, Blockchain Explorers, Wallets, Marketplaces.
+
+### Implementation Plan
+
+1. Publish open source reference implementation and instructions related to the creation, storage and reading of royalty utxos.
+2. Implement in open source libraries and tooling such as [Lucid](https://github.com/spacebudz/lucid), [Blockfrost](https://github.com/blockfrost/blockfrost-backend-ryo), etc.
+3. Achieve additional "buy in" from existing community actors and implementors such as: blockchain explorers, token marketplaces, minting platforms, wallets.
+
+## References
```

Comment:
The CIP-0001 format, and therefore any parsers that may work on CIP material, has no _References_ section.  I would recommend putting these links inline: with the first use of a CIP term linked to its document & further references unlinked (like Wikipedia).

There is a format where anchors can be collected at the bottom of the page but I can't find a reference to it or a name for it... but some CIPs use it (e.g. CIP-0094) and you could model your linking on them.  Defining the links in that way would collect them at the bottom of the page which would help serve the purpose of your list here.

Reply:
I wasn't aware of that syntax, very cool. Updated as recommended.